exercises/joseph_ramsay/ex02_oo_basics/src/shapes.h

```diff
@@ -0,0 +1,5 @@
+#include "circle.h"
```

Comment:
don't create a combined header file. I can understand the appeal, but it is an unnecessary layer which does not provide any additional abstraction